students/18th/team2/wanglee/account/views.py

```diff
@@ -0,0 +1,58 @@
+import json
+from django.views import View
+from django.http  import JsonResponse
+from .models      import User
+
+class UserSignup(View):
+    def post(self, request):
+        data     = json.loads(request.body)
+        email    = data['email']
+        password = data['password']
+        username = data['username']
+        phone    = data['phone']
+        temps = User.objects.filter(username=username) | User.objects.filter(email=email) | User.objects.filter(phone=phone)
+
+        if email == '' or password == '':
+            return JsonResponse({"message":"KEY_ERROR"}, status=400)
+
+        elif '@' not in email or '.' not in email:
```

Comment:
이 부분의 조건문은 아래와 같이 조금 더 간결하게 만들어줄 수 있습니다!
```suggestion
        elif ('@' or '.') not in email:
```